config/crd/bases/image.toolkit.fluxcd.io_imagerepositories.yaml

```diff
@@ -483,16 +483,24 @@ spec:
                 description: LastScanResult contains the number of fetched tags.
                 properties:
                   latestTags:
+                    description: |-
+                      LatestTags is a small sample of the tags found in the last scan.
+                      It's the first 10 tags when sorting all the tags in descending
+                      alphabetical order.
                     items:
                       type: string
                     type: array
+                  revision:
+                    description: Revision is a stable hash of the scanned tags.
+                    type: string
                   scanTime:
+                    description: ScanTime is the time when the last scan was performed.
                     format: date-time
                     type: string
                   tagCount:
+                    description: TagCount is the number of tags found in the last
+                      scan.
                     type: integer
-                required:
-                - tagCount
                 type: object
```

Comment:
Hmm it's probably better if we don't add `+optional` to this field now... I'll revert the change only for this one,